RenameClockEntry is renamed to TryRenameClockEntry

This causes renode fail to run using the latest version:

```
xous-core/emulation/peripherals/BetrustedEcI2C.cs(174,33): error CS1061: Type `Antmicro.Renode.Time.IClockSour
ce' does not contain a definition for `RemoveClockEntry' and no extension method `RemoveClockEntry' of type `Antmicro.Renode.Time.
IClockSource' could be found. Are you missing an assembly reference?
```

--- a/emulation/peripherals/BetrustedEcI2C.cs
+++ b/emulation/peripherals/BetrustedEcI2C.cs
@@ -171,7 +171,7 @@ namespace Antmicro.Renode.Peripherals.I2C.Betrusted
         private void FinishTransaction()
         {
             // this.Log(LogLevel.Error, "I2C: Removing clock entry for {0}",this.irqTimeoutCallback);
-            machine.ClockSource.RemoveClockEntry(FinishTransaction);
+            machine.ClockSource.TryRemoveClockEntry(FinishTransaction);
             irqTimeoutCallbackQueued = 0;
             if (shouldSendTxRxIrq)
             {

--- a/emulation/peripherals/BetrustedSocI2C.cs
+++ b/emulation/peripherals/BetrustedSocI2C.cs
@@ -162,7 +162,7 @@ namespace Antmicro.Renode.Peripherals.I2C.Betrusted
 
         private void FinishTransaction()
         {
-            machine.ClockSource.RemoveClockEntry(FinishTransaction);
+            machine.ClockSource.TryRemoveClockEntry(FinishTransaction);
             if (shouldSendTxRxIrq)
             {
                 shouldSendTxRxIrq = false;

